packages/astro/test/astro-sync.test.js

```diff
@@ -1,6 +1,7 @@
 import assert from 'node:assert/strict';
 import * as fs from 'node:fs';
 import { before, describe, it } from 'node:test';
+import ts from 'typescript';
```

Comment:
I think this is fine, since it is only a test, but I'll let other comment, if we want to import the full `typescript` package here.

Reply:
This is fine and is the typical way to import from TS